src/Xrefcheck/Verify.hs

```diff
@@ -150,8 +152,11 @@ instance Buildable VerifyError where
       "   Use of such anchors is discouraged because referenced object\n\
       \   can change silently whereas the document containing it evolves.\n"
 
-    ExternalResourceInvalidUri ->
-      "⛂  Invalid URI\n"
+    ExternalResourceInvalidUri err ->
+      "⛂  Invalid URI (" +| err |+ ")\n"
+
+    ExternalResourceUriConversionError err ->
+      "⛂  Invalid URI (" +| displayException err |+ ")\n"
```

Comment:
It seems `displayException @ParserExceptionBs` is [implemented](https://hackage.haskell.org/package/modern-uri-0.3.4.4/docs/src/Text.URI.Types.html#line-255) via `errorBundlePretty`, which is likely to be a multi-line string.

In that case, instead of surrounding the message in `(` `)`, I think it would be better to display it in its own line, with indentation:

```suggestion
      unlinesF
        [ "⛂  Invalid URI"
        , indentF 4 $ build $ displayException err
        ]
```